src/garage/envs/metaworld_task_names_wrapper.py

```diff
@@ -0,0 +1,58 @@
+"""A wrapper for Metaworld environment to expose names of all tasks."""
+import gym
+
+
+class MetaworldTaskNamesWrapper(gym.Wrapper):
```

Comment:
Can we just add this to metaworld? It looks pretty useful generally.

Reply:
i agree -- we can just add this feature to metaworld

Reply:
Will do.

Reply:
`MetaworldTaskNamesWrapper` moved to here https://github.com/rlworkgroup/metaworld/pull/62